WordPressUtils/src/main/java/org/wordpress/android/util/ImageUtils.java

```diff
@@ -647,7 +647,11 @@ public static Bitmap getVideoFrameFromVideo(String videoPath, int maxWidth, Map<
             // I've see this kind of error on one of my testing device
             AppLog.e(AppLog.T.MEDIA, "The passed video path is invalid: " + videoPath);
         } finally {
-            mediaMetadataRetriever.release();
+            try {
+                mediaMetadataRetriever.release();
+            } catch (Exception e) {
+                AppLog.e(AppLog.T.MEDIA, "The passed video path is invalid: " + videoPath);
```

Comment:
I'm wondering if the potential exception produced when releasing the `MediaMetadataRetriever` would be related to the video path, is this accurate? Alternatively, should we include the message coming from the exception instead?

Reply:
Good point.  Updated

Reply:
It doesn't look related to the path. 


> @throws IOException When an {@link IOException} is thrown while closing a {@link
> MediaDataSource} passed to {@link #setDataSource(MediaDataSource)}. This throws clause exists
> since API {@link android.os.Build.VERSION_CODES#TIRAMISU}, but this method can throw in
> earlier API versions where the exception is not declared.
> https://android.googlesource.com/platform/frameworks/base/+/master/media/java/android/media/MediaMetadataRetriever.java#1097

The updated exception message (`AppLog.e(AppLog.T.MEDIA, "Failed to release mediaMetadataRetriever.", e);`) looks better to me.